spec/factories/foods.rb

```diff
@@ -0,0 +1,10 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :food do
+    name { Faker::Food.unique.ingredient }
+    measurement_unit { Faker::Food.measurement }
+    price { Faker::Number.decimal(l_digits: 2, r_digits: 2) }
+    user { FactoryBot.create :user }
```

Comment:
I see. So this is what you meant about not using traits. And I see all tests pass so, maybe it's unnecessary. I'll search for it tomorrow so we can decide which to keep for our merge conflict in users